### PR TITLE
Correct TLS configuration

### PR DESCRIPTION
**Pull request by mjibson**

### CODE REVIEW

**Comment by juliusv** [comment]
@mjibson Thanks so much! Could you reopen this PR against the `next-release` branch instead of `master`? We'll merge that once 0.16.0 is out. At some point we could offer a multi-version docs site, but for now, one has to view https://github.com/prometheus/docs/tree/next-release/content/docs for any breaking changes that aren't in the latest full release version yet.

**Comment by juliusv** [comment]
God, you even opened it against `next-release`. Maybe I shouldn't be doing this at night after a long work day anymore :)

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -185,6 +182,24 @@ Where `<scheme>` may be `http` or `https` and `<path>` is a valid URL path.
 regex `[a-zA-Z_][a-zA-Z0-9_-]`.
 
 
+### TLS configuration `<tls_config>`
+
+A `tls_config` allows configuring TLS connections.
+
+```
+
```
Comment (juliusv) [inline content/docs/operating/configuration.md]: Remove the extra empty line here.

**Comment by juliusv** [comment]
:+1: otherwise, thanks! /cc @jimmidyson who contributed those features to take a last look.

**Comment by jimmidyson** [comment]
:+1: 

Sorry I missed updating the docs when I added this :(

**Comment by mjibson** [comment]
Remove the extra line.

**Comment by juliusv** [comment]
Thanks :)

**Pull request merged**